OrbitGl/App.cpp

```diff
@@ -860,7 +861,12 @@ void OrbitApp::UpdateModuleList(int32_t pid) {
         LoadModulesFromPreset(process, preset);
         Capture::GSessionPresets = nullptr;
       }
-      // To this point ----------------------------------
```

Comment:
This was a terminator for TODO comment :)

Reply:
Ups.  Sorry. Readded it again.